client/src/components/recipes-collections/RecipeListCard.jsx

```diff
@@ -0,0 +1,23 @@
+import recipeIcon from "./menu.png";
+
+const RecipeListCard = ({recipe, viewRecipe, navToSingleRecipeView}) => {
+  return (
+    <div
+      className="recipe-list-card-container"
+      onClick={() => navToSingleRecipeView(recipe.id)}>
+      <div className="recipe-list-card-img">
+        <img
+          src={!recipe.image ? recipe.image : recipeIcon}
```

Comment:
Is this logic correct?

Reply:
It's not! I intentionally made it show the default image for now, but I'll open a [question](https://techtonicaorg.slack.com/archives/C03GQEAPTLP/p1668488428628779) in tech-help to see if I can get the logic right. Thanks for catching that!